### PR TITLE
Form components Design QA

### PR DESCRIPTION
**Pull request by spark3-mdb**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [Name of ticket](https://jira.mongodb.org/browse/[name-of-ticket])

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Style change (purely visual updates; if this is the case, attach a screenshot below!)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

## Feedback Levels

Please try to mark your comments with one of these levels

1. Praise (👏 🚀 🔥 🍾)
2. Opinions, Ideas, Suggestions (💡 👀)
3. Let’s discuss @ Code review (🤷 🤔 🧐 )
4. Doesn’t meet agreed upon standards (😬 ⚠️ ‼️)
5. Missing an edge case (🛑 🙅 ❌)
6. Missing core functionality (⛔️ ☣️ 🚧 🌋 💀)

(Note: Use ⌘-⌃-␣ to open the Emoji keyboard!)

### CODE REVIEW

File: packages/expandable-card/src/ExpandableCard.tsx [hunk 1]
```diff
@@ -254,7 +255,7 @@ const ExpandableCard = ({
         className={summaryStyle}
         onClick={onClick}
         onKeyPress={onClick}
-        tabIndex={0}
+        tabIndex={-1}
```
Comment (TheSonOfThomp) [inline packages/expandable-card/src/ExpandableCard.tsx:258]: 🤔  Should we just remove the `role=button` attribute?

File: packages/expandable-card/src/ExpandableCard.tsx [hunk 2]
```diff
@@ -269,6 +270,7 @@ const ExpandableCard = ({
               glyph="ChevronUp"
               size={24}
               fill={palette.gray.base}
+              tabIndex={0}
```
Comment (TheSonOfThomp) [inline packages/expandable-card/src/ExpandableCard.tsx:273]: ⚠️ This should be an `IconButton` to have the right hover & focus styles

File: packages/form-footer/src/FormFooter.tsx [hunk 3]
```diff
@@ -122,7 +122,7 @@ export default function FormFooter({
             variant="default"
             onClick={onBackClick}
             className={buttonStyle}
-            leftGlyph={<ChevronLeftIcon />}
+            leftGlyph={<ArrowLeftIcon />}
```
Comment (TheSonOfThomp) [inline packages/form-footer/src/FormFooter.tsx:125]: 👍

**Review by TheSonOfThomp — CHANGES_REQUESTED** [review]

File: packages/expandable-card/src/ExpandableCard.tsx (hunk 2 above)
Comment (spark3-mdb) [reply packages/expandable-card/src/ExpandableCard.tsx:273]: This was going to be a separate ticket, but I guess I can just handle it here. I'll get to this tomorrow.

File: packages/expandable-card/src/ExpandableCard.tsx [hunk 4]
```diff
@@ -264,12 +267,15 @@ const ExpandableCard = ({
...
               className={cx(iconStyle, iconTransitionStyle[state])}
-              glyph="ChevronUp"
-              size={24}
-              fill={palette.gray.base}
-            />
+              aria-label={title}
+              tabIndex={0}
+            >
+              <Icon glyph="ChevronUp" size={24} fill={palette.gray.base} />
+            </IconButton>
```
Comment (TheSonOfThomp) [inline packages/expandable-card/src/ExpandableCard.tsx:278]: 👍  Just played around with VoiceOver and I have just one nit: the `aria-label` should be something like "expand/collapse card". Having the title there again is a little confusing

**Review by TheSonOfThomp — APPROVED** [review]

**Pull request merged**